source/darwin/dispatch_queue_event_loop_private.h

```diff
@@ -0,0 +1,55 @@
+#ifndef AWS_IO_DARWIN_DISPATCH_QUEUE_H
+#define AWS_IO_DARWIN_DISPATCH_QUEUE_H
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <Security/Security.h>
+#include <aws/common/mutex.h>
+#include <aws/common/thread.h>
+#include <aws/io/tls_channel_handler.h>
+#include <dispatch/dispatch.h>
+
+struct dispatch_loop;
+struct dispatch_loop_context;
+
+struct dispatch_loop {
+    struct aws_allocator *allocator;
+    dispatch_queue_t dispatch_queue;
+    struct aws_task_scheduler scheduler;
+    struct aws_event_loop *base_loop;
+
+    /*
+     * Internal ref-counted dispatch loop context to processing Apple Dispatch Queue Resources.
+     * The context keep track of the live status of the dispatch loop. Dispatch queue should be
+     * nulled out in context when it is cleaned up.
+     */
+    struct dispatch_loop_context *context;
+
+    /* Synced data handle cross thread tasks and events, and event loop operations*/
+    struct {
+        /**
+         * The lock is used to protect synced_data across the threads. It should be acquired whenever we touched the
```

Comment:
Trivial: "It should be acquired whenever data in the synced_data struct is accessed or modified." ?